src/common/common-core/src/main/java/ee/ria/xroad/common/SystemProperties.java

```diff
@@ -125,6 +125,18 @@ private SystemProperties() {
     public static final String PROXY_UI_API_ACME_CHALLENGE_PORT_ENABLED =
             PREFIX + "proxy-ui-api.acme-challenge-port-enabled";
 
+    public static final String PROXY_UI_API_ACME_CERTIFICATE_RENEWAL_ACTIVE =
+            PREFIX + "proxy-ui-api.acme-certificate-renewal-active";
+
+    public static final String PROXY_UI_API_ACME_CERTIFICATE_RENEWAL_RETRY_DELAY =
+            PREFIX + "proxy-ui-api.acme-certificate-renewal-retry-delay";
+
+    public static final String PROXY_UI_API_ACME_CERTIFICATE_RENEWAL_INTERVAL =
+            PREFIX + "proxy-ui-api.acme-certificate-renewal-interval";
+
+    public static final String PROXY_UI_API_ACME_RENEWAL_TIME_BEFORE_EXPIRATION_DATE =
+            PREFIX + "proxy-ui-api.acme-renewal-time-before-expiration-date";
```

Comment:
All other renewal related properties have `acme-certificate-renewal` prefix. I think it would be good to use common prefix for all renewal related properties. Either `acme-certificate-renewal` or `acme-renewal` would suite.